src/components/Footer/Footer.scss

```diff
@@ -24,6 +24,12 @@
     margin: 0 5px;
   }
 
+  .wkp-footer-left {
+    a {
+      white-space: nowrap;
```

Comment:
Why that ?

Reply:
TO avoid having a button two lined content e.g.:
'Developer
 Portal'

Reply:
if I remove it nothing happens

Reply:
ah yes it happens :)